tests/test_checker/test_noqa.py

```diff
@@ -24,6 +24,9 @@
 _PY38 = (3, 8) <= sys.version_info < (3, 9)
 _PY39 = (3, 9) <= sys.version_info < (3, 10)
 
+# Versions for different platform-specific fixtures.
+_WIN = os.name == 'nt'
```

Comment:
See https://github.com/wemake-services/wemake-python-styleguide/blob/master/wemake_python_styleguide/logic/system.py#L11

Reply:
And https://github.com/wemake-services/wemake-python-styleguide/blob/master/wemake_python_styleguide/logic/system.py#L3

Reply:
Done